src/search-headless.ts

```diff
@@ -411,6 +411,9 @@ export default class SearchHeadless {
     const { limit, offset, sortBys, locationRadius } = this.state.vertical;
     const { referrerPageUrl, context } = this.state.meta;
     const { userLocation } = this.state.location;
+    const zeroOffset = this.state.vertical.offset ?? 0;
+    const inputIsPreviousQuery = input == this.state.query.mostRecentSearch;
```

Comment:
nit, can you access `mostRecentSearch` and `queryId` on line 409?

Reply:
Done!

Reply:
yup, adjusted